text/0265-event-vision.md

```diff
@@ -0,0 +1,74 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to generic logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from generic logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and generic logs,
+and encourage the use of events over generic logs.
+
+### Interoperability with generic logging libraries
+
+It should be possible to send events from the Event API to a generic logging library (e.g. Log4j).
+This allows users to integrate events from the Event API into an existing (non-OpenTelemetry) log stream.
+
+Note: If a user chooses to send events from the Event API to a generic logging library, and they have
+also chosen to send the logs from their generic logging library to the OpenTelemetry Logging SDK, then they should
+avoid sending events from the Event API directly to the OpenTelemetry Logging SDK since that would lead to duplicate
```

Comment:
How we envision this working? Will events also leverage log bridges or will events be produced by the SDK (e.g., like metrics) and just happen to use the same otlp log exporter that log bridges use? This is a bit confusing to be honest.

Reply:
I'm trying not to overspecify in the OTEP. The key part of the proposal is that Events should go to the Logging SDK (where they will go through Logging Processors -> Logging Exporters).